kani_metadata/src/harness.rs

```diff
@@ -24,6 +24,8 @@ pub struct HarnessMetadata {
     pub goto_file: Option<PathBuf>,
     /// The `#[kani::<>]` attributes added to a harness.
     pub attributes: HarnessAttributes,
+
+    pub contracts: Vec<String>,
```

Comment:
Can you please rename this to `reachable_contracts` or something similar? I think the name is a bit misleading. As I mentioned offline, we might want to add to the harness which contract to verify.